src/collections/_documentation/workflow/notifications/workflow.md

```diff
@@ -3,12 +3,99 @@ title: 'Workflow'
 sidebar_order: 2
 ---
 
-Most activity within Sentry will generate a workflow notification. Core workflow notifications include:
+Most activity within Sentry will generate a workflow notification. You can also configure various personal notifications from your [personal notifications settings](https://sentry.io/settings/account/notifications) and any specific project's settings in the Project Settings page.
```

Comment:
same here - can we link to https://sentry.io/settings/account/notifications/workflow/ ?